app/assets/stylesheets/application.scss

```diff
@@ -59,24 +55,70 @@
   color: $red;
 }
 
+.e-table-cell,
+.e-table-title {
+  border-bottom: 1px solid $grey-3;
+  padding-bottom: 3px;
+  color: $black;
+}
+
+.e-table-cell {
+  font-weight: normal;
+}
+
+.enhanced-checkbox {
+  background-color: $grey-3;
+  clear: left;
+  padding: 5px;
+  margin-left: 55px;
+  margin-bottom: 20px;
+  display: none;
+}
+
+.enhanced-labels {
+  margin: 2px;
+}
+
 .form-control {
   border: 2px solid $black;
   border-radius: 0;
   font-size: 19px;
   line-height: 1.31579;
 }
 
+.form-group {
+  padding-bottom: 15px;
+}
+
 .form-label {
   @include heading-24;
   display: block;
   float: left;
   clear: left;
+  font-family: "nta", Arial, sans-serif;
+  font-weight: 400;
+  text-transform: none;
+  font-size: 16px;
+  line-height: 1.25;
 }
 
 .form-label input:checked ~ .control-indicator:after {
   display: block;
 }
 
+.grid-row {
+  @extend %grid-row;
+
+  .ab-search-wrapper-left {
+    clear: both;
+    @include grid-column( 1 / 2 );
+  }
+
+  .ab-search-wrapper-right {
+    @include grid-column( 1 / 2 );
```

Comment:
I know we spoke a while ago about not doing this as even though it is in all the CSS docs, it not how companies actually use it.

Can we create a new card which is `govukify the CSS`

Reply:
For internal apps I'd use bootstrap components including the grid.

Reply:
will make the card